apps/i18n/studio/en_us.json

```diff
@@ -285,6 +285,7 @@
   "playSoundWinPoint": "play win point sound",
   "playSoundWinPoint2": "play win point 2 sound",
   "playSoundWood": "play wood sound",
+  "playSoundR2D2Random": "play R2-D2 random sound",
```

Comment:
sorting should not be an issue, but it might look a little odd in the dropdown:
play `random R2-D2` sound
play `R2-D2 1` sound
play `R2-D2 2` sound

I figured that since we qualify the sound instance after the object for the other cases, we should do the same for the range entry. But when the random entry is selected, I can see your point that it would read better. (much like it would be better to say `play R2-D2 sound 1`!)